includes/cancel-on-next-payment-date.php

```diff
@@ -167,10 +171,7 @@ function pmproarc_conpd_pmpro_change_level( $level, $user_id, $old_level_status,
  *
  * @return string The updated translated text.
  */
-function pmproarc_conpd_gettext_cancel_text( $translated_text, $text, $domain ) {
-	if ( function_exists( 'pmproconpd_gettext_cancel_text' ) ) {
-		return $translated_text;
-	}
+function pmproconpd_gettext_cancel_text( $translated_text, $text, $domain ) {
```

Comment:
We should add a check above this to ensure it's not already defined - some people have both plugins installed.

Reply:
I see the code below handles the loading of this file, just in case and to be extra safe it might still be worth it to do a function_exists check.

Reply:
Decision made in Slack that this check is not necessary.